MachUpskillingFY17/JabbR-Core
# Kevin/authorize redirect

Fixes Issue #253
- Updated AspNetCore dependencies to use 1.10-* packages for consistency. (Advised from the product team)
- The issue was a package conflict with the current 1.1.0-* Authentication package and a 1.0.0 version of IISIntegration. `"Microsoft.AspNetCore.Server.IISIntegration": "1.1.0-preview1-final"` resolves this.
- Startup.cs authentication and identity flow is correct.

**To Test:**
1. Rebuild project with updated project.json
2. Build project
3. Observe the beautiful Login page that loads on your screen.
4. Double check you are hitting Home Controller by navigating to your root and ensure you are redirected again.
